benchmarks/component_rendering.py

```diff
@@ -3,18 +3,27 @@
 from django.template import Context, Template
 from django.test import override_settings
 
-from django_components import component
+from django_components import component, types
 from django_components.middleware import CSS_DEPENDENCY_PLACEHOLDER, JS_DEPENDENCY_PLACEHOLDER
 from tests.django_test_setup import *  # NOQA
 from tests.testutils import BaseTestCase, create_and_process_template_response
 
 
 class SlottedComponent(component.Component):
-    template_name = "slotted_template.html"
+    template: types.django_html = """
+        {% load component_tags %}
```

Comment:
I moved the template files into Python, and added syntax highlighting with `types.django_html` where possible.